src/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
```

Comment:
- [Optional] My default, most Ruby programmers brings all private methods to the bottom and public to the top. This convention allows someone to easily work on your codebase.